crates/ere-dockerized/build.rs

```diff
@@ -0,0 +1,65 @@
+use build_utils::{detect_sdk_version, detect_self_crate_version};
+use std::{env, fs, path::Path};
+
+fn main() {
+    generate_ere_version();
```

Comment:
Thought not for this PR: maybe at some point it would be good to also append the git hash, since some people could be using `master` and could be useful?